src/entities/models/dto.py

```diff
@@ -33,6 +33,16 @@ class FilingTaskStateDTO(BaseModel):
     change_timestamp: datetime | None = None
 
 
+class ContactInfoDTO(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    id: int | None = None
+    first_name: str
+    last_name: str
+    email: str
+    phone: str
+
+
 class FilingDTO(BaseModel):
```

Comment:
same thing as the dao.

Reply:
doesn't look like this is resolved yet? I still see contact_info as removed from the DTO.